rusk/src/bin/config/chain.rs

```diff
@@ -62,4 +64,8 @@ impl ChainConfig {
     pub(crate) fn migration_height(&self) -> Option<u64> {
         self.migration_height
     }
+
+    pub(crate) fn generation_timeout(&self) -> Option<Duration> {
+        self.generation_timeout
```

Comment:
Could you add this new config field in default.config.toml? Also an comment / doc explaining its purpose, recommended values etc